tests/functional_tests/transfer.py

```
@@ -36,6 +36,17 @@
 from framework.daemon import Daemon
 from framework.wallet import Wallet
 
+def diff_transfers(actual_transfers, expected_transfers, ignore_order = True):
```

Ruff (0.15.20)

[warning] Boolean default positional argument in function definition

(FBT002)